portfolio/src/main/webapp/script.js

```diff
@@ -102,3 +103,27 @@ function createListElement(text) {
   liElement.innerText = text;
   return liElement;
 }
+
+/**
+ * Formats the comment json.
+ * @param {json} json Comment object in json form.
+ * @return {string} A formatted string.
+ */
+function formatComment(json) {
+    let name = json.name;
+    let comment = json.comment;
+
+    let year = json.time.date.year;
+    let month = json.time.date.month;
+    let day = json.time.date.day;
+
+    let hour = json.time.time.hour;
+    let minute = json.time.time.minute;
+    let second = json.time.time.second;
+
+    let jsonString = "Name: " + name + "\n"
```

Comment:
Is there a better way to format date time?

Reply:
Fixed